controllers/metric.go

```diff
@@ -25,22 +25,6 @@ var (
 		},
 		[]string{"name"},
 	)
-	plcTempsProcessSecondsCounter = prometheus.NewCounterVec(
```

Comment:
IIRC, these metrics were originally for our own investigations in performance, so I think we're totally fine removing them :+1: